public/restaurant.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Document</title>
+      <link href="../main.css" rel="stylesheet">
+      <script src="https://kit.fontawesome.com/9a9c81b339.js" crossorigin="anonymous"></script>
+      <link rel="stylesheet" href="https://unpkg.com/leaflet@1.6.0/dist/leaflet.css"
+            integrity="sha512-xwE/Az9zrjBIphAcBb3F6JVqxf46+CDLwfLMHloNu6KEQCAWi6HcDUbeOfBIptF7tcCzusKFjFw2yuvEpDL9wQ=="
+            crossorigin="" />
+      <script src="https://unpkg.com/leaflet@1.6.0/dist/leaflet.js"
+            integrity="sha512-gZwIG9x3wUXg2hdXF6+rVkLF/0Vi9U8D2Ntg4Ga5I5BZpVkVxlJWbSQtXPSiUTtC0TjtGOmxa1AJPuV0CPthew=="
+            crossorigin=""></script>
+</head>
+
+<body>
+
+      <!--  Header section  -->
+      <div id="header">
+            <a href="../index.html">
+                  <h1>Yelpington</h1>
+            </a>
+      </div>
+      <div class="middleContainer">
+            <!--  Map  -->
+            <div id='map'></div>
+            <!--  SideBar  -->
+            <nav>
+                  <ul>
+                        <style>
+                              #restId {
+                                    display: none;
+                              }
+                        </style>
```

Comment:
This will work right here, but you should bring this style tag up inside the head tag, or you could add this rule to your css file